packages/core/EventManager.ts

```diff
@@ -146,8 +147,8 @@ export default class EventManager {
         meetingId: createdEventObj ? createdEventObj.id : result.createdEvent?.id?.toString(),
         meetingPassword: createdEventObj ? createdEventObj.password : result.createdEvent?.password,
         meetingUrl: createdEventObj ? createdEventObj.onlineMeetingUrl : result.createdEvent?.url,
-        externalCalendarId: evt.destinationCalendar?.externalId,
-        credentialId: evt.destinationCalendar?.credentialId,
+        externalCalendarId: isCalendarType ? evt.destinationCalendar?.externalId : undefined,
+        credentialId: result.credentialId ?? evt.destinationCalendar?.credentialId,
```

Comment:
The thing here it was saving calendar Credential ID, when there are some cases that video references need their right value.